src/interfaces/IKintoAppRegistry.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+interface IKintoAppRegistry {
+    /* ============ Structs ============ */
+
+    struct Metadata {
+        address developerWallet; // 160 bits
```

Comment:
what do you think about renaming `developerWallet` to just `admin`?

Reply:
done